sdk/trace/span.go

```diff
@@ -298,6 +316,26 @@ func typeStr(i interface{}) string {
 	return fmt.Sprintf("%s.%s", t.PkgPath(), t.Name())
 }
 
+func recordStackTrace(err error) string {
```

Comment:
```suggestion
func recordStackTrace() string {
```

It doesn't look like `err` is ever used in this function.